package/MDAnalysis/coordinates/XDR.py

```diff
@@ -41,6 +41,24 @@ def offsets_filename(filename, ending='npz'):
                                                         ending=ending))
 
 
+def read_numpy_offsets(fname):
+    """read offsets into a dictionary
+
+    Parameters
+    ----------
+    fname : str
+        filename of offsets
+
+    Returns
+    -------
+    offsets: dict
+        dict with offsets
+    """
+    with open(fname) as f:
```

Comment:
only to `open` `np.load` doesn't have argument for that. So I assume it will deal with it internally.